### PR TITLE
✨ Added the first page

### PR DESCRIPTION
**Pull request by DNaunin**
Added the welcome page
Added two grids to show the design as desired

### CODE REVIEW

File: components/messages/Message.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { Story, Meta } from "@storybook/react/types-6-0";
+
+export default {
+  title: "Common/Pages",
+  component: Welcome,
+} as Meta;
+
+const Template: Story = (args) => <Welcome {...args} />;
+
+export const firstWelcome = Template.bind({});
```
Comment (fre-ben) [inline components/messages/Message.stories.tsx:13]: Is there a secondWelcome? Maybe name it something like "splashPage"?

File: components/messages/Welcome.module.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.welcomecontainer {
+  display: grid;
+  grid-template-columns: 2fr, 1fr;
+  grid-template-rows: repeat(4, 1fr);
+  align-content: center;
+  align-items: center;
```
Comment (fre-ben) [inline components/messages/Welcome.module.css:6]: It looks like these lines don't effect the layout

File: components/messages/Welcome.module.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.title {
+  font-family: Ranchers, sans-serif;
+  color: var(--main-color);
+  text-align: center;
+  font-size: 1.8rem;
+  align-self: end;
+  grid-area: 1 / 1 / 2 / 2;
```
Comment (fre-ben) [inline components/messages/Welcome.module.css:15]: The grid-area seems to have no effect

File: components/messages/Welcome.module.css [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.subtitle {
+  font-family: Roboto, sans-serif;
+  color: var(--main-color);
+  text-align: center;
+  font-size: 1rem;
+  grid-area: 2 / 1 / 3 / 2;
```
Comment (fre-ben) [inline components/messages/Welcome.module.css:23]: The grid-area seems to have no effect

**Review by fre-ben — COMMENTED** [review]
LGTM 🧙 
Just check some css properties, which seem to have no effects on the layout :)

**Review by RonjaMeinecke — APPROVED** [review]
Great progress 💥

**Pull request merged**